xtrlock.c

```diff
@@ -36,38 +36,102 @@
 #include <math.h>
 #include <ctype.h>
 #include <values.h>
+#include <unistd.h>
+#include <stdbool.h>
 
 #ifdef SHADOW_PWD
 #include <shadow.h>
 #endif
 
+//#define DEBUG
+#ifdef DEBUG
+#define debug_print(...) do{{printf(__VA_ARGS__);} }while (0)
+#else
+#define debug_print(...) do{}while (0)
+#endif
+
 Display *display;
-Window window, root;
+Window window, blank_window, trans_window, root;
```

Comment:
I don't think I created that variable. It seems to be coming from the previous developers and I didn't really looked at it before since there wasn't really any warning of unused variables from the compiler (probably because it's global). Since I don't really know what did the previous developers want by defining it, I wonder should I really just remove it because it feels like throwing away stuffs that are not mine. 

Anyway, should I just remove any variables that's created by other developers for unknown purpose because it has not been used yet?